config/routes.rb

```diff
@@ -3,6 +3,7 @@
   resources :todos, only: [:index, :create, :update] do
     collection do
       delete :delete_all, to: 'todos#delete_all'
+      get :sort, to: 'todos#sort'
```

Comment:
Okay ver stiys yral, baich enqanel lav varaint chi.

Duzy kinar ver `index` methodumy prosto veradardznis, arandzin Route petkchi.
By default orinak tox `ABC` ini sort, isk shtex petq ini `CBA`..

Baich qani ver logikat `PaginateItemsService` -in yiraya sarqat uran poxely shat jamanak k tane, tak chto tox sti mna.